activejob/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Allow `Sidekiq` and `que` to report the job id back to `ActiveJob::Base` as
```

Comment:
While we're here, are there any other adapters that need this? resque?

On Thu, May 7, 2015 at 3:58 PM, Jeroen van Baarsen <notifications@github.com
> wrote:

> In activejob/CHANGELOG.md
> <https://github.com/rails/rails/pull/20056#discussion_r29889786>:
>
> > @@ -1,3 +1,8 @@
> > +*   Allow `Sidekiq` and `que` to report the job id back to `ActiveJob::Base` as
>
> @rafaelfranca <https://github.com/rafaelfranca> Yes sounds good.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails/rails/pull/20056/files#r29889786>.
>



-- 
*Kevin D. Deisz*
*TrialNetworks* - part of DrugDev
Software Developer
383 Elliot Street, Suite G
Newton, MA 02464
+1 617.952.4071 x134 (office)
+1 703.615.0396 (mobile)
kdeisz@trialnetworks.com